src/transport/smtp/async_transport.rs

```diff
@@ -47,7 +70,16 @@ impl Tokio1Transport for AsyncSmtpTransport<Tokio1Connector> {
 
     /// Sends an email
     async fn send_raw(&self, envelope: &Envelope, email: &[u8]) -> Result<Self::Ok, Self::Error> {
+        #[cfg(any(
+            not(feature = "tokio1-pool"),
+            any(feature = "tokio02", feature = "async-std1")
+        ))]
         let mut conn = self.inner.connection().await?;
+        #[cfg(all(
+            feature = "tokio1-pool",
+            not(any(feature = "tokio02", feature = "async-std1"))
+        ))]
+        let mut conn = self.inner.get().await?;
 
         let result = conn.send(envelope, email).await?;
 
```

Comment:
`conn.quit()` shouldn't be called for connections taken from the pool (I can't seem to be able to select the correct line, but it's the one under this comment)

Reply:
Fixed!